docs/_scripts/install_from_signed_repository.sh

```diff
@@ -2,7 +2,7 @@
 
 BASE_URL=$(http "$BASE_ADDR""$DISTRIBUTION_HREF" | jq -r '.base_url')
 BASE_PATH=$(http "$BASE_ADDR""$DISTRIBUTION_HREF" | jq -r '.base_path')
-PUBLIC_KEY_URL="$BASE_URL"/repodata/public.key
+PUBLIC_KEY_URL="$BASE_URL"/repodata/repomd.xml.key
```

Comment:
I'd rather not use a name that nobody else uses ("public.key").  "repomd.xml.asc" is a SUSE-ism, but it makes the most sense of the various conventions I've seen IMO.

All the repos I've seen in the RH ecosystem either don't do signing at all, or don't include the key along with the repo at all, or use semi-difficult-to-guess names.  The closest I've seen to a proper convention is CentOS 7 using "RPM-GPG-KEY-$reponame".

Reply:
The .asc is the signature file, but I think this key is the one used during signing to produce that signature.

Reply:
Whoops, I made a typo.  I meant "repomd.xml.key"

You are correct

Reply:
@dalley sorry, you're correct, my apologies. .asc is an ascii-armored sig-doc, which can be either attached to the thing-being signed or a separate signature if generated with --detach-sig - which is the use in this case.  .sig is binary-format signature.

Reply:
I believe the `.asc` file is a just the detached signature, there should be no key in it.

Reply:
No problem, cryptography is super confusing and I barely know what I'm doing :)